docs/getting-started/networks.md

```diff
@@ -0,0 +1,153 @@
+# Networks
+
+There are a variety of ways to setup either local or remote networks with automation, detailed below.
+All the required files are found in the [networks directory](https://github.com/cosmos/cosmos-sdk/tree/develop/networks) and additionally the `local` or `remote` sub-directories.
+
+## Local Testnet
+
+From the [networks/local directory](https://github.com/cosmos/cosmos-sdk/tree/develop/networks/local):
+
+### Requirements
+
+- [Install gaia](https://cosmos.network/docs/getting-started/installation.html)
+- [Install docker](https://docs.docker.com/engine/installation/)
+- [Install docker-compose](https://docs.docker.com/compose/install/)
+
+### Build
+
+Build the `gaiad` binary and the `tendermint/gaiadnode` docker image.
+
+Note the binary will be mounted into the container so it can be updated without
+rebuilding the image.
+
+```
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+
+# Build the linux binary in ./build
+make build-linux
+
+# Build tendermint/gaiadnode image
+make build-docker-gaiadnode
+```
+
+### Run a testnet
+
+To start a 4 node testnet run:
+
+```
+make localnet-start
+```
+
+The nodes bind their RPC servers to ports 26657, 26660, 26662, and 26664 on the host.
+This file creates a 4-node network using the gaiadnode image.
+The nodes of the network expose their P2P and RPC endpoints to the host machine on ports 26656-26657, 26659-26660, 26661-26662, and 26663-26664 respectively.
+
+To update the binary, just rebuild it and restart the nodes:
+
+```
+make build-linux
+make localnet-stop
+make localnet-start
+```
+
+### Configuration
+
+The `make localnet-start` creates files for a 4-node testnet in `./build` by calling the `gaiad testnet` command.
+
+The `./build` directory is mounted to the `/gaiad` mount point to attach the binary and config files to the container.
+
+For instance, to create a single node testnet:
+
+```
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+
+# Clear the build folder
```

Comment:
I'd put this as steps outside the code square and only de bash commands with ``

Reply:
i think it's fine the way it is. Actually kinda like big code blocks like this in documentation and find there isn't enough of it